src/ashell/jerry-code.c

```diff
@@ -326,8 +326,17 @@ void restore_zjs_api() {
 #ifdef BUILD_MODULE_SENSOR
     zjs_sensor_init();
 #endif
+    jerry_value_t global_obj = jerry_get_global_object();
```

Comment:
should just use ZVAL here, so you don't have to free it, this is a weird place to set this also, shouldnt this be in the zjs_modules_init() function?